sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -778,5 +778,11 @@ private[sql] class SQLConf extends Serializable with CatalystConf with Logging {
   def clear(): Unit = {
     settings.clear()
   }
+
+  def reset(): Unit = {
+    sqlConfEntries.values.asScala.foreach { entry =>
```

Comment:
Ok, I see. It will remove all the user-set parameters. Let me do it now. Thanks!